mne/io/meas_info.py

```diff
@@ -1866,6 +1889,29 @@ def _force_update_info(info_base, info_target):
             i_targ[key] = val
 
 
+def _add_timedelta_to_meas_date(meas_date, dt):
+    """Add a timedelta to a meas_date tuple.
+
+    Parameters
+    ----------
+    meas_date : tuple | None
+        The Info object you want to use for overwriting values
+        in target Info objects.
+    dt : datetime.timedelta
```

Comment:
can you rename dt to td. dt suggests it is datetime.datatime and not datetime.timedelta

Reply:
or better delta_t to match naming below

Reply:
@bloyl can you just take care of this one?

Reply:
should be done now